medical_manufacturer/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Medical Manufacturer",
+    "summary": "Extends companies with concept of medical manufacturers.",
```

Comment:
This doesn't extend companies. A `res.company` in Odoo is a very different concept from a `res.partner` that is tagged as a company. 

Probably `Extension of medical`, like the readme says